### PR TITLE
KDS-1751-kaio-migration-search-slider

### PR DESCRIPTION
**Pull request by gyfchong**
Slider + SearchField

### CODE REVIEW

File: packages/components/src/InputRange/_docs/InputRange.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { InputRange } from "@kaizen/components"'
+/>
+
+## Overview
+
+A range of things.
```
Comment (JakePitman) [inline packages/components/src/InputRange/_docs/InputRange.mdx:23]: Should we describe what kind of thing? Or is it totally open to the consumer

File: packages/components/src/Slider/_docs/Slider.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <div className="flex flex-col gap-16">
+      <Slider {...args} labelText="Default" variant="default" />
+      <Slider {...args} labelText="Prominent" variant="prominent" />
+    </div>
+  ),
+}
+
+export const MinMaxLabels: Story = {
+  render: () => (
+    <Slider
```
Comment (JakePitman) [inline packages/components/src/Slider/_docs/Slider.stories.tsx:41]: Are we able to use `args` instead of `render` for this one?

Comment (gyfchong) [reply packages/components/src/Slider/_docs/Slider.stories.tsx:41]: True! Nice catch

File: packages/components/src/InputRange/_docs/InputRange.mdx (hunk 1 above)
Comment (gyfchong) [reply packages/components/src/InputRange/_docs/InputRange.mdx:23]: Up to them!

**Review by JakePitman — APPROVED** [review]

**Pull request merged**